update/assembly.rst

```diff
@@ -2,13 +2,19 @@
 Updating Assemblies
 ===================
 
-Updates to assemblies can be performed through `Webin-CLI <../submit/general-guide/webin-cli.html>`_, and result in the
+Updates to assemblies can be performed through `Webin-CLI <../submit/general-guide/webin-cli.html>`_,  and for all except SARS-CoV-2 assemblies*, result in the
```

Comment:
I first looked at the bottom of the page for it and did not find it immediately - might help to add a link on the 'SARS-CoV-2 assemblies*' text to the information 
e.g.
:ref:`SARS-CoV-2 assemblies* <SC2 update>`. 

.. _SC2 update:
*\*To maximise the speed of processing,.....

there is an example on the ORCID claiming guide where this is used for 'navigation box' https://github.com/enasequence/read_docs/commit/9337306b09bd6b6d0d77648977d7cf50f99cd20b